ablator/utils/base.py

```diff
@@ -241,11 +241,9 @@ def init_weights(module: nn.Module):
 
     Notes
     -----
-    - If the module is a Linear layer, initialize weight values from a normal distribution N(mu=0, std=1.0).
-    If biases are available, initialize them to zeros.
+    - If the module is a Linear layer, initialize weight values from a normal distribution N(mu=0, std=1.0). If biases are available, initialize them to zeros.
```

Comment:
@hieuchi911 in general it is not a good idea to mix PRs. e.g. fixing docs and writing tutorials.

Reply:
Do I need to separate this to a new branch?